src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  height: 60px;
+  display: flex;
+  align-items: center;
+
+  position: relative;
+
+  font-size: 12px;
+  font-weight: $medium-font;
+  text-transform: uppercase;
+  white-space: nowrap;
+  text-decoration: none;
+  text-align: center;
+  color: $primary-text-color;
+}
+
+.nav__link:hover {
+  color: $main-color;
+}
+
+.is-active {
+  color: $main-color;
+}
```

Comment:
The `.is-active` class selector should be nested within `.nav__link` using the `&` operator to maintain SCSS nesting conventions. This helps to keep the CSS structure clear and shows that `.is-active` is a modifier of `.nav__link`.